### PR TITLE
Create stub of justin russell bio

### PR DESCRIPTION
**Pull request by jrussell2**
:warning: Work in Progress :warning: 

Silly @rwnilsen3 and @marandamir debating who is more ruling than the other. Those of us that are truly awesome don't have to always announce it. We just exude it.

My bio is bland. What else should I add.

### CODE REVIEW

File: bios/justin-russell.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#Justin Russell
+* I'm not interesting
```
Comment (jrussell2) [inline bios/justin-russell.md]: Ain't that the truth

Comment (mgerde) [inline bios/justin-russell.md]: Do you have any children?

Comment (mgerde) [inline bios/justin-russell.md]: Maybe they are more interesting.  :smiley:

**Comment by jrussell2** [comment]
Lol. Yes, @mgerde, they are more interesting.

**Pull request merged**